.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Acme e2e Test CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Seup Node.js '14.x'
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Prepare artifacts (e.g. extensions) for docker-compose stack
+        run: mvn --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -DskipTests verify --file pom.xml
+      - name: Build the docker-compose stack
+        run: |
+          export USER=`id -u`
+          export GROUP=
+          echo "Running as ${USER}:${GROUP}"
+          java bin/prerequisites.java
```

Comment:
Perhaps we can rename `prerequisitis.java` to something like `envcheck.java` as it is simpler?